mix_helpers.exs

```diff
@@ -704,7 +704,12 @@ defmodule Mix.Appsignal.Helper do
   end
 
   if System.otp_release() >= "23" do
-    defp tls_options, do: [versions: [:"tlsv1.3", :"tlsv1.2"]]
+    defp tls_options do
```

Comment:
If we change this, I still need to test and review it in more detail, we also need to change it in the diagnose report transmission.

Reply:
Added the same change to `Appsignal.Transmitter` as well.